testScripts/compareVersions/compareWithVLFeat.sh.in

```diff
@@ -0,0 +1,353 @@
+#!/bin/bash
+
+CMD_VLF=@VLFEAT@
+CMD_POP=@CMAKE_RUNTIME_OUTPUT_DIRECTORY@/popsift-demo
+CMD_AWK=@AWK@
+CMD_SORT=@SORT@
+CMD_GP=@GNUPLOT@
+CMD_CONVERT=@CONVERT@
+CMD_COMPARE=@CMAKE_RUNTIME_OUTPUT_DIRECTORY@/popsift-compareSiftFiles
+DIR_AWK=@CMAKE_CURRENT_SOURCE_DIR@/awk
+DIR_TEST=@PopSift_TESTFILE_PATH@
+
+VLFEAT_DEFAULT_PARAMETERS=""
+
+# VLFeat multiplies the descriptors with 512 before converting to uchar,
+# so we use 9 for a 2**9 multiplier as well.
+POPSIFT_DEFAULT_PARAMETERS="--pgmread-loading --norm-mode=classic --norm-multi 9 --write-as-uchar --write-with-ori --initial-blur 0.5 --sigma 1.6 --threshold 0 --edge-threshold 10.0"
+
+# FILES="hash level1 boat"
+# FILES="level1"
+# FILES="hash"
+# FILES="boat1 boat"
+# FILES="boat1"
+FILES="boat"
```

Comment:
I'm getting errors of files not found with this scrpt.
I think the error come from the ambiguity of `FILES`: here it seems that it contains the dataset(s) that we want to use, but almost everywhere in the script is used to name the file to process, e.g `boat.pgm` instead of `boat/img1.pgm`.
Is the script supposed to run on each and every image of (possibly) multiple datasets? If so I think it needs some work i.e. it misses a nested `for img in img1 img2 img3....` almost everywhere